config/default/manager_audit_patch.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: audit
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        volumeMounts:
+        - mountPath: /certs
+          name: cert
+          readOnly: true
+      volumes:
+      - name: cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-server-cert
```

Comment:
It looks like audit would need cert rotator if running standalone? Is that something users are able to configure? Or do we only enable it if a webhook is running?

Reply:
> It looks like audit would need cert rotator if running standalone?

That's right. I tried enabling cert-controller for both audit and webhook deployment, and I encountered weird issues so this PR assumes that the audit deployment will always rely on the cert generated by webhook's cert-controller.

What I can do is I can get rid of the requirement in https://github.com/open-policy-agent/gatekeeper/blob/master/main.go#L189 and allow the audit deployment to run cert-controller. By default, we will enable cert-controller for webhook and disable it for audit. This way users will have the option to run standalone audit deployment and enable cert-controller. Does that sound good?

Reply:
SGTM!